addons/sale/models/sale.py

```diff
@@ -1073,6 +1073,18 @@ def product_id_change(self):
 
         result = {'domain': domain}
 
+        name = product.name_get()[0][1]
+        if product.description_sale:
+            name += '\n' + product.description_sale
+
+        vals.update(name=name)
```

Comment:
minimal changes again

vals['name'] = name

Reply:
@moylop260

Could you review this PR again, please?

Regards